src/hydat/parse.ts

```diff
@@ -1,26 +1,18 @@
-export function isAlpha(c: string) {
-  const regex = new RegExp('^[A-Za-z]$');
-  return regex.test(c);
-}
+const isAlpha = RegExp.prototype.test.bind(/^[A-Za-z]$/);
+const isDigit = RegExp.prototype.test.bind(/^[0-9]$/);
 
-export function isDigit(c: string) {
-  const regex = new RegExp('^[0-9]$');
-  return regex.test(c);
-}
+const isAlDig = (c: string) => isAlpha(c) || isDigit(c);
```

Comment:
この型は推論できるはずなのでなくてもよい
```suggestion
const isAlDig = (c) => isAlpha(c) || isDigit(c);
```

Reply:
推論できませんでした……（謎）．
「any なのでダメ」と出ます（出ませんか？）．